src/pages/ShareCodePage.tsx

```diff
@@ -46,13 +72,24 @@ function ShareCodePage({report}: ShareCodePageProps) {
                 shouldShowBackButton
             />
             <ScrollView style={[themeStyles.flex1, themeStyles.pt3]}>
-                {/* 
+                <View style={[themeStyles.workspaceSectionMobile, themeStyles.ph5]}>
+                    {/* 
                     Right now QR code download button is not shown anymore
                     This is a temporary measure because right now it's broken because of the Fabric update.
                     We need to wait for react-native v0.74 to be released so react-native-view-shot gets fixed.
                     
                     Please see https://github.com/Expensify/App/issues/40110 to see if it can be re-enabled.
                 */}
+                    <QRShare
+                        ref={qrCodeRef}
+                        url={url}
+                        title={title}
+                        subtitle={subtitle}
+                        logo={isReport ? expensifyLogo : (UserUtils.getAvatarUrl(currentUserPersonalDetails?.avatar, currentUserPersonalDetails?.accountID) as ImageSourcePropType)}
+                        logoRatio={CONST.QR.DEFAULT_LOGO_SIZE_RATIO}
+                        logoMarginRatio={CONST.QR.DEFAULT_LOGO_MARGIN_RATIO}
```

Comment:
One more question: It looks like we previously had this:

```suggestion
                        logoRatio={isReport ? CONST.QR.EXPENSIFY_LOGO_SIZE_RATIO : CONST.QR.DEFAULT_LOGO_SIZE_RATIO}
                        logoMarginRatio={isReport ? CONST.QR.EXPENSIFY_LOGO_MARGIN_RATIO : CONST.QR.DEFAULT_LOG
```

Reply:
Fixed!